packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -73,8 +73,12 @@
   "force_source_retrieve_text": "SFDX: Retrieve Source from Org",
   "force_source_retrieve_this_source_text":
     "SFDX: Retrieve This Source from Org",
+  "force_source_retrieve_using_manifest_text":
+    "SFDX: Retrieve Source from Org using Manifest",
   "force_source_deploy_text": "SFDX: Deploy Source to Org",
   "force_source_deploy_this_source_text": "SFDX: Deploy This Source to Org",
+  "force_source_deploy_using_manifest_text":
+    "SFDX: Deploy Source to Org using Manifest",
```

Comment:
@ruthemmanuelle These are two new commands that are specific to source deploy/retrieve using the manifest file. I'd just taken the existing commands and added "using Manifest".

Reply:
Since "using" is more than four characters long, it should be capitalized even though it's a preposition. But maybe "SFDX: Retrieve Source in Manifest from Org" and "SFDX: Deploy Source in Manifest to Org" would be better? That way they'd be a bit more similar to the existing "... This Source ..." deploy/retrieve commands.

Reply:
Adding @allileong to this. "Deploy Source in Manifest" isn't really grammatically correct. You're deploying source identified by the manifest there's no actual source in the manifest. Initially I'd thought about Deploy Source With Manifest but that wasn't correct either. You're using the manifest to identify what source you're retrieving or deploying which is why using made sense.

Reply:
I like "Using" better since it does seem the most accurate, but I think both are good options, and the user would likely understand how to use the command in either case.

Reply:
Well, the source is "in" the manifest in the same sense in which chapters in a book could be "in" a table of contents. But I'm ok with "SFDX: Deploy Source to Org Using Manifest", since you guys think it's better. Just make sure that you capitalize "Using".

Reply:
@JimSuplizio - I know I said in my last comment that I was ok with the "...Using Manifest" option, but I really think that "SFDX: Retrieve Source in Manifest from Org" and "SFDX: Deploy Source in Manifest to Org" are better. Since @ntotten liked my last comment, I assume he does too? 

We want the user to be able to grasp the purpose of the command as quickly as possible while they're scanning the command palette, and putting "Manifest" earlier in the message would help with that. The "from Org"/"to Org" part doesn't distinguish these commands from the others.

And, if users don't know what a manifest is then using "in Manifest" would help them understand that the types to deploy/retrieve are listed in the manifest. 

"SFDX: Retrieve Source in Manifest from Org" and "SFDX: Deploy Source in Manifest to Org" also sound better (that is, they read more smoothly) than "SFDX: Retrieve Source from Org Using Manifest" and "SFDX: Deploy Source to Org Using Manifest".

Reply:
@ruthemmanuelle I'll change them to "SFDX: Retrieve Source in Manifest from Org" and "SFDX: Deploy Source in Manifest to Org"